pages/api/contact.js

```diff
@@ -0,0 +1,31 @@
+export default function (req, res) {
+  const nodemailer = require('nodemailer')
+  const transporter = nodemailer.createTransport({
+    // This is for mailtrap tester. Need to update to real SMTP settings, such as Gmail.
+    host: 'smtp.mailtrap.io',
+    port: 2525,
+    auth: {
+      user: process.env.EMAIL_USERNAME,
+      pass: process.env.EMAIL_PASSWORD
```

Comment:
For consistency, I'd also update these to `SMTP_USERNAME` and `SMTP_PASSWORD`

Reply:
Done